lib/ClimaLandSimulations/src/ClimaLandSimulations.jl

```diff
@@ -17,6 +17,7 @@ using ClimaLand.Canopy
 using ClimaLand.Canopy.PlantHydraulics
 import ClimaLand.Parameters as LP
 import ClimaComms
+context = ClimaComms.context()
```

Comment:
usually the context is set by the driver file (so experiments/ozark.jl here) rather than in src/

Reply:
^^ yes, because I think it will change if on CPU or CPU

Reply:
If `include("Fluxnet/fluxnet_simulation.jl")` is called in the driver it can be there, but otherwise it needs to be in the driver itself. It's not good practice to include a src/ file in the driver. A good solution could be to define a function `get_context` in fluxnet_simulation.jl which returns `ClimaComms.context()`, then call that from the driver. Would that be okay with you?

Reply:
I am not sure what you mean by the driver?

Reply:
I just mean wherever the simulation is actually getting setup and run, so ozark.jl in this case I think